apitesters/offerings.ts

```diff
@@ -53,6 +53,7 @@ function checkPackage(pack: PurchasesPackage) {
 function checkOffering(offering: PurchasesOffering) {
   const identifier: string = offering.identifier;
   const serverDescription: string = offering.serverDescription;
+  const metadata: Map<string, any> = offering.metadata;
```

Comment:
for the type, I'm not sure about this, but I think metadata is optional, so should we type when it does not exist? ` | undefined` or similar?

Reply:
It's optional but we made it default to empty dictionary/map in the native SDKs because much easier to handle for users 🤷‍♂️ 

Here is the mapper from hybrid commong which is where this `metadata` gets its value from! This screenshot shows that metadata is a non-nullable dictionary. So `metadata` in the React Native layer here should always be here 💪 
<img width="637" alt="Screenshot 2023-06-06 at 2 21 18 PM" src="https://github.com/RevenueCat/react-native-purchases/assets/401294/24a83518-5fb2-4562-9fd9-a5d03701f570">

Reply:
Yeah this is good. No need to differentiate between "no metadata" and "empty metadata" IMO.

Reply:
Typing seems to be wrong... afaik it should be Record, not Map in typescript.